runtime/lua/vim/_editor.lua

```diff
@@ -43,6 +43,8 @@ for k, v in pairs({
   vim._submodules[k] = v
 end
 
+vim.json = {}
```

Comment:
Hmm did it not work in the meta file?

Reply:
It generates a documentation entry for `vim.json.json`

Reply:
But this doesn't work either lol

Reply:
Lol you keep getting scalded by our _top quality_ doc generation.

For now you could try adding `--- @nodoc` above . That might work.

Reply:
Yeah _someone_ should really rewrite the doc generator.